src/native/log.rs

```diff
@@ -0,0 +1,96 @@
+use pyo3::pymodule;
+
+#[pymodule]
+pub mod logger {
+    use pyo3::exceptions::PyValueError;
+    use pyo3::prelude::*;
+    use pyo3::types::PyDict;
+
+    use datadog_log::logger::{
+        logger_configure_file, logger_configure_std, logger_disable_file, logger_disable_std,
+        logger_set_log_level, FileConfig, LogEventLevel, StdConfig, StdTarget,
+    };
+
+    #[pyfunction]
+    #[pyo3(signature = (**kwds))]
+    fn configure(kwds: Option<&Bound<'_, PyDict>>) -> PyResult<()> {
+        let output;
+
+        if let Some(kwargs) = kwds {
+            output = kwargs
+                .get_item("output")?
+                .ok_or_else(|| PyValueError::new_err("Missing output argument"))?
+                .extract()?;
+        } else {
+            output = "output".to_string();
+        }
+
+        match output.as_str() {
+            "stdout" => logger_configure_std(StdConfig {
+                target: StdTarget::Out,
+            })
+            .map_err(|e| PyValueError::new_err(e.to_string())),
+            "stderr" => logger_configure_std(StdConfig {
+                target: StdTarget::Err,
+            })
+            .map_err(|e| PyValueError::new_err(e.to_string())),
+            "file" => {
+                let kwargs =
+                    kwds.ok_or_else(|| PyValueError::new_err("Missing arguments for file"))?;
+
+                let path: String = kwargs
+                    .get_item("path")?
+                    .ok_or_else(|| {
+                        PyValueError::new_err("Missing required argument for file: path")
+                    })?
+                    .extract()?;
+
+                let max_files: u64 = kwargs
+                    .get_item("max_files")?
+                    .map(|v| v.extract())
+                    .transpose()?
+                    .unwrap_or(0);
+
+                let max_size_bytes: u64 = kwargs
+                    .get_item("max_size_bytes")?
+                    .map(|v| v.extract())
+                    .transpose()?
+                    .unwrap_or(0);
+
+                let cfg = FileConfig {
+                    path,
+                    max_files,
+                    max_size_bytes,
+                };
+                logger_configure_file(cfg).map_err(|e| PyValueError::new_err(e.to_string()))
+            }
+            other => Err(PyValueError::new_err(format!("Invalid output: {other}"))),
+        }
+    }
+
+    /// Disable logging output by type: "file", "stdout", "stderr"
+    #[pyfunction]
+    fn disable(output: &str) -> PyResult<()> {
```

Comment:
can't we use enums (or string enums) here?

Throwing a runtime error while this can be dealt during compile time looks bad. 

Same for the `set_log_level` API.

Reply:
IIRC the Python compiler does not perform any type checking when the code is transformed to bytecode so I guess no matter if is invalid value (enum) or string, the error will be shown at runtime. Aside from that, I'm not that well versed in Python so this might not be the most idiomatic way to craft the API. I've chosen that path because it seemed like a simple module and it's kind of convenient to configure it from env vars without the need to do translations between strings to enums.

Reply:
I've added the documentation.